xc7/primitives/slicem/Ndram/dpram64.pb_type.xml

```diff
@@ -4,9 +4,9 @@
 
   <!-- A1 - Read port -->
   <input  name="A"   num_pins="6" />
-  <output name="O6"  num_pins="1" />
```

Comment:
Why change this name?  O6 is correct?

Reply:
There's only one output for DPRAM32/64, so I renamed it to O. The {A-D}_DRAMs still use the names from the CLB diagram.

Reply:
If the names in the next level up use the CLB diagram names, I'm okay.